FirstAppBun/CustomCell.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  customCell.swift
+//  FirstAppBun
+//
+//  Created by David Alex Marcu on 26.07.2022.
+//
+
+import UIKit
+
+class CustomCell: UITableViewCell {
+
+    
+    @IBOutlet var ImageView2: UIImageView!
+    @IBOutlet var TextLabel2: UILabel!
+    
+    override func awakeFromNib() {
```

Comment:
If you are not using the override functions, please remove them. They are automatically called by iOS.